File: app/src/main/java/org/d3ifcool/jagosholat/views/activities/TentangKamiActivity.java
package org.d3ifcool.jagosholat.views.activities;

/**
 * Created by Faisal Amir
 * FrogoBox Inc License
 * =========================================
 * 40-mobpro-4002-jago-sholat
 * Copyright (C) 05/09/2018.
 * All rights reserved
 * -----------------------------------------
 * Name     : Muhammad Faisal Amir
 * E-mail   : faisalamircs@gmail.com
 * Line     : bullbee117
 * Phone    : 081357108568
 * Majors   : D3 Teknik Informatika 2016
 * Campus   : Telkom University
 * -----------------------------------------
 * id.amirisback.frogobox
 */
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

import org.d3ifcool.jagosholat.R;

public class TentangKamiActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tentang_kami);

        // -----------------------------------------------------------------------------------------
        ImageView imageIcan = findViewById(R.id.imageican);
        ImageView imageBryan = findViewById(R.id.imagebrian);
        ImageView imageAmir = findViewById(R.id.imageamir);
        // -----------------------------------------------------------------------------------------
        Picasso.get().load(R.drawable.about_us_amir).into(imageAmir);
        Picasso.get().load(R.drawable.about_us_ican).into(imageIcan);
        Picasso.get().load(R.drawable.about_us_bryan).into(imageBryan);
        // -----------------------------------------------------------------------------------------

    }
}

File: app/src/main/java/org/d3ifcool/jagosholat/ui/activitiy/TentangKamiActivity.kt
package org.d3ifcool.jagosholat.ui.activitiy

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.squareup.picasso.Picasso
import kotlinx.android.synthetic.main.activity_tentang_kami.*
import org.d3ifcool.jagosholat.R

/**
 * Created by Faisal Amir
 * FrogoBox Inc License
 * =========================================
 * 40-mobpro-4002-jago-sholat
 * Copyright (C) 05/09/2018.
 * All rights reserved
 * -----------------------------------------
 * Name     : Muhammad Faisal Amir
 * E-mail   : faisalamircs@gmail.com
 * Line     : bullbee117
 * Phone    : 081357108568
 * Majors   : D3 Teknik Informatika 2016
 * Campus   : Telkom University
 * -----------------------------------------
 * id.amirisback.frogobox
 */
class TentangKamiActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tentang_kami)
        // -----------------------------------------------------------------------------------------
        Picasso.get().load(R.drawable.about_us_amir).into(imageican)
        Picasso.get().load(R.drawable.about_us_ican).into(imageican)
        Picasso.get().load(R.drawable.about_us_bryan).into(imagebrian)
        // -----------------------------------------------------------------------------------------
    }
}